### PR TITLE
DID Documents section editorial re-review

### PR DESCRIPTION
**Pull request by grantnoble**
- Line length changes
- Wording, grammar, and punctuation changes
- Removal of superfluous "Example" intros
- Other misc changes

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/did-core/pull/141.html" title="Last updated on Jan 7, 2020, 12:56 AM UTC (d566763)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/141/601e0d9...grantnoble:d566763.html" title="Last updated on Jan 7, 2020, 12:56 AM UTC (d566763)">Diff</a>

### CODE REVIEW

**Comment by grantnoble** [comment]
Merge conflicts resolved.

File: index.html [hunk 1]
```diff
@@ -1138,57 +1138,56 @@ <h1>
...
       </p>
 
+      <dl>
+        <dt><dfn>@context</dfn></dt>
+        <dd>
+The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
+where the value of the first <a>URI</a> is
+<code>https://www.w3.org/ns/did/v1</code>. If more than one <a>URI</a> is
+provided, the <a>URIs</a> MUST be interpreted as an ordered set. It is
+RECOMMENDED that dereferencing the <a>URIs</a> results in a document containing
```
Comment (TallTed) [inline index.html]:
```suggestion
RECOMMENDED that dereferencing each <a>URI</a> result in a document containing
```

File: index.html [hunk 2]
```diff
@@ -1305,76 +1302,78 @@ <h2>
...
+The value of the <code>id</code> property, if present, MUST be a <a>URI</a>. The
+array of public keys MUST NOT contain multiple entries with the same
+<code>id</code>. In this case, a <a>DID document</a> processor MUST produce an
+error.
       </p>
 
       <p>
-The value of the <code>type</code> property MUST be exactly one public key value
-(see available key types in Appendix <a href="#registries"></a>).
+The value of the <code>type</code> property MUST be exactly one public key
```
Comment (TallTed) [inline index.html]:
```suggestion
The value of the <code>type</code> property MUST be exactly one public key type.
```

File: index.html [hunk 3]
```diff
@@ -1305,76 +1302,78 @@ <h2>
...
+value. For a registry of available key types and formats, see Appendix
```
Comment (TallTed) [inline index.html]:
```suggestion
For a registry of available key types and formats, see Appendix
```

File: index.html [hunk 4]
```diff
@@ -1305,76 +1302,78 @@ <h2>
...
+to the revocation information for the key (for example, a revocation list). Each
+<a>DID method</a> specification is expected to detail how revocation is
+performed and tracked.
       </p>
 
       <p>
-All public key formats MUST be expressed in either JSON Web Key (JWK)
-format using the <code>publicKeyJwk</code> property or one of the
-formats listed in the table below. Public key expression MUST NOT use any other key format.
+All public key formats MUST be expressed in either JSON Web Key (JWK) format
```
Comment (TallTed) [inline index.html]:
```suggestion
Public keys of all types MUST be expressed in either JSON Web Key (JWK) format
```

File: index.html [hunk 5]
```diff
@@ -1498,70 +1498,64 @@ <h2>
...
         </li>
 
         <li>
 If <em>result</em> does not contain at least the <code>id</code>,
-<code>type</code>, and <code>controller</code> properties as well as any
-mandatory public cryptographic material, as determined by the
-<em>result</em>'s <code>type</code> property, throw an error.
+<code>type</code>, and <code>controller</code> properties, as well as any
+mandatory public cryptographic material, as determined by the <code>type</code>
+property of <em>result</em>, throw an error.
```
Comment (TallTed) [inline index.html]:
```suggestion
property of the <em>result</em>, throw an error.
```

File: index.html [hunk 6]
```diff
@@ -1633,13 +1626,16 @@ <h2>
...
-Authentication as explained in Section <a href="#authentication"></a>.
-This is particularly important for key
-recovery in the case of key loss, when the subject no longer has
+on their behalf.
+      </p>
+
+      <p class="note">
+Authorization is separate from authentication, as explained in
+Section <a href="#authentication"></a>. This is particularly important for key
+recovery, in the case of key loss when the subject no longer has
```
Comment (TallTed) [inline index.html]:
```suggestion
recovery, in the case of key loss, when the subject no longer has
```

File: index.html [hunk 7]
```diff
@@ -1648,28 +1644,23 @@ <h2>
...
+Delegation, which may be layered:
       </p>
 
-      <ol>
+      <ul>
         <li>
-A <a>DID registry</a> could implement a coarse
-grained <code>controller</code> pattern by enabling <a>DID documents</a> to
-express the <a>DID</a> of another <a>DID controller</a> that controls it, or
+A <a>DID registry</a> could implement a coarse grained <code>controller</code>
```
Comment (TallTed) [inline index.html]:
```suggestion
A <a>DID registry</a> could implement a coarse-grained <code>controller</code>
```

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Updated, but left it as "results". "result" just didn't sound right to me.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 4 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 5 above)
Comment (grantnoble) [reply index.html]: I don't agree with this change. All the preceding steps refer to _result_ as a parameter (e.g., "Set _result_ to _value_.", and so on).

File: index.html (hunk 6 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 7 above)
Comment (grantnoble) [reply index.html]: Yep.

File: index.html [hunk 8]
```diff
@@ -1913,45 +1893,41 @@ <h2>
...
       </ul>
 
       <p>
-This approach to data modeling is often called an "open world
-assumption", meaning that anyone can say anything about any other
-thing. This approach often feels in conflict with building simple
-and predictable software systems. Balancing extensibility with
-program correctness is always more challenging with an open world
-assumption than it is with closed software systems.
+This approach to data modeling is often called an <em>open worldassumption</em>,
```
Comment (msporny) [inline index.html]:
```suggestion
This approach to data modeling is often called an <em>open world assumption</em>,
```

**Comment by msporny** [comment]
Editorial, multiple positive reviews, changes applied, merging.

File: index.html (hunk 5 above)
Comment (TallTed) [reply index.html]: Ah, OK.

**Pull request merged**